src/datasources/balances-api/balances-api.manager.ts

```diff
@@ -40,11 +43,31 @@ export class BalancesApiManager implements IBalancesApiManager {
     this.zerionBalancesApi = zerionBalancesApi;
   }
 
-  async getBalancesApi(chainId: string): Promise<IBalancesApi> {
+  async getBalancesApi(
+    chainId: string,
+    safeAddress: `0x${string}`,
+  ): Promise<IBalancesApi> {
     if (this.zerionChainIds.includes(chainId)) {
       return this.zerionBalancesApi;
     }
 
+    try {
+      const transactionApi =
+        await this.transactionApiManager.getTransactionApi(chainId);
+      await transactionApi.getSafe(safeAddress);
```

Comment:
Nit: I'd add a comment _why_ we're doing this.

Reply:
Added in [32c52a2](https://github.com/safe-global/safe-client-gateway/pull/1615/commits/32c52a21ac656ee3375d237d82204c5a0fca7c61), please let me know if you find it not clear enough 🙂